yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
you did `yarn install` on the root folder when there's nothing there so it produced this file

Reply:
ping @chayG5